packages/caravan-fees/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@caravan/fees",
+  "version": "0.0.0",
+  "description": "Utility library for fee bumping bitcoin transactions",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsup src/index.ts --format cjs,esm --dts",
+    "dev": "npm run build -- --watch",
+    "test": "jest src",
+    "test:watch": "jest --watch src"
+  },
+  "keywords": [
+    "bitcoin",
+    "cpfp",
+    "rbf",
+    "feebumping",
+    "blockchain"
+  ],
+  "author": "",
+  "license": "MIT",
+  "engines": {
+    "node": ">=20"
+  },
+  "devDependencies": {
+    "@caravan/eslint-config": "*",
+    "tsconfig": "*"
+  },
+  "dependencies": {
+    "@caravan/bitcoin": "*"
```

Comment:
Make sure to declare all dependencies here otherwise it won't work if installed outside of the monorepo. E.g. BigNumber library, bitcoinjs-lib etc.

Reply:
So I need to update all the dependencies in the next commit

Reply:
[updated](https://github.com/caravan-bitcoin/caravan/pull/114/files#diff-a3e31745046775327f41608ad7eccffc05717d319e1f0fb67360040d4488fd61)